src/backend/base/pyproject.toml

```diff
@@ -198,6 +204,11 @@ ignore = [
     "ISC001", # Messes with the formatter
 ]
 
+[tool.ruff.lint.per-file-ignores]
+"langflow/api/v1/*" = [
+    "TCH", # FastAPI needs to evaluate types at runtime
```

Comment:
I had to ignore here. Ideally we would use `flake8-type-checking.runtime-evaluated-decorators` but I couldn't make it work with fastapi's APIRouter annotations without importing the router from a separate file. This looks like a bug in ruff.